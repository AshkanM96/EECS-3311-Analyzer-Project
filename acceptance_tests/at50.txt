disjoin
equals
sum
open_set_enumeration
integer_constant (2)
integer_constant (3)
integer_constant (-5)
integer_constant (0)
close_set_enumeration
negative
integer_constant (-0)
boolean_constant (false)
evaluate
type_check
evaluate
