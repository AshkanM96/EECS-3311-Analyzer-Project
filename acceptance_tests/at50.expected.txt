  Expression currently specified: ?
  Report: Expression is initialized.
->disjoin
  Expression currently specified: (? || nil)
  Report: OK.
->equals
  Expression currently specified: ((? = nil) || nil)
  Report: OK.
->sum
  Expression currently specified: (((+ ?) = nil) || nil)
  Report: OK.
->open_set_enumeration
  Expression currently specified: (((+ {?}) = nil) || nil)
  Report: OK.
->integer_constant(2)
  Expression currently specified: (((+ {2, ?}) = nil) || nil)
  Report: OK.
->integer_constant(3)
  Expression currently specified: (((+ {2, 3, ?}) = nil) || nil)
  Report: OK.
->integer_constant(-5)
  Expression currently specified: (((+ {2, 3, -5, ?}) = nil) || nil)
  Report: OK.
->integer_constant(0)
  Expression currently specified: (((+ {2, 3, -5, 0, ?}) = nil) || nil)
  Report: OK.
->close_set_enumeration
  Expression currently specified: (((+ {2, 3, -5, 0}) = ?) || nil)
  Report: OK.
->negative
  Expression currently specified: (((+ {2, 3, -5, 0}) = (- ?)) || nil)
  Report: OK.
->integer_constant(0)
  Expression currently specified: (((+ {2, 3, -5, 0}) = (- 0)) || ?)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: (((+ {2, 3, -5, 0}) = (- 0)) || False)
  Report: OK.
->evaluate
  Expression currently specified: (((+ {2, 3, -5, 0}) = (- 0)) || False)
  Report: True
->type_check
  Expression currently specified: (((+ {2, 3, -5, 0}) = (- 0)) || False)
  Report: (((+ {2, 3, -5, 0}) = (- 0)) || False) is type-correct.
->evaluate
  Expression currently specified: (((+ {2, 3, -5, 0}) = (- 0)) || False)
  Report: True
